Library/Homebrew/cask/lib/hbc/artifact/installer.rb

```diff
@@ -23,7 +23,7 @@ module ScriptInstaller
         def install_phase(command: nil, **_)
           ohai "Running #{self.class.dsl_key} script '#{path.relative_path_from(cask.staged_path)}'"
           FileUtils.chmod "+x", path unless path.executable?
-          command.run(path, **args, path: PATH.new(HOMEBREW_PREFIX/"bin", HOMEBREW_PREFIX/"sbin", ENV["PATH"]))
+          command.run!(path, **args, env: { "PATH" => PATH.new(HOMEBREW_PREFIX/"bin", HOMEBREW_PREFIX/"sbin", ENV["PATH"]) })
```

Comment:
Good catch!

I wonder whether we could have caught this earlier with an `artifact/installer_spec.rb`.
I feel this change warrants creating that spec, and have the class under test start out with `PATH=/var/dummy`, then assert that the system command finds both `HOMEBREW_PREFIX/"bin"` and `/var/dummy` in its environment `PATH`.

(No full coverage needed for now – just the tiny bit that’s being changed here!)